app/scripts/models/reliers/oauth.js

```diff
@@ -31,7 +31,9 @@ define([
       // would have redirectUri as its base.
       redirectTo: null,
       // whether to fetch and derive relier-specific keys
-      keys: false
+      keys: false,
+      // always redirect to the rp, useful during email verification signup flow
+      always: false
```

Comment:
> Have other alternatives been considered, like making use of redirectTo or allowing the relier to specify an URL other than the redirectURI?

`redirectTo` is conventionally used by reliers to dynamically change the redirect URL, which we don't support now but haven't ruled out for the future. Let's not conflate that with configuring the UX around redirection.